<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArduinoProject</class>
 <widget class="QDialog" name="ArduinoProject">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="7">
    <widget class="QLabel" name="prsL">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="8">
    <widget class="QProgressBar" name="humidityBar">
     <property name="minimumSize">
      <size>
       <width>35</width>
       <height>0</height>
      </size>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QLabel" name="hmdtL">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Temp</string>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Hmdt</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Prs</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QProgressBar" name="pressureBar">
     <property name="minimumSize">
      <size>
       <width>35</width>
       <height>0</height>
      </size>
     </property>
     <property name="minimum">
      <number>720</number>
     </property>
     <property name="maximum">
      <number>800</number>
     </property>
     <property name="value">
      <number>720</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QProgressBar" name="tempBar">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>35</width>
       <height>0</height>
      </size>
     </property>
     <property name="minimum">
      <number>-30</number>
     </property>
     <property name="maximum">
      <number>40</number>
     </property>
     <property name="value">
      <number>-30</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="tempL">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_5">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QLabel" name="label_6">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Управление</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QCheckBox" name="led">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>LED</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="comfort">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Таблица комфорта</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QPushButton" name="startRec">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <strikeout>false</strikeout>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>Начать запись</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QPushButton" name="stopRec">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Завершить запись</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="stats">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Отключено</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Статус:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QPushButton" name="readSD">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Прочитать запись</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="personalization">
        <property name="text">
         <string>Персонализация</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="instructions">
        <property name="text">
         <string>Инструкция</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Подключение</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="portlist">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="conB">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Подключить</string>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="disconnectB">
        <property name="text">
         <string>Отключить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
